### PR TITLE
Add "miner_uptime" metric

### PR DESCRIPTION
**Pull request by warren-ru**

### CODE REVIEW

File: trex/gauges.go [hunk 1]
```diff
@@ -8,6 +8,11 @@ import (
 var nameSpace = "trex"
 
 var (
+	Uptime = promauto.NewGaugeVec(prometheus.GaugeOpts{
```
Comment (rluisr) [inline trex/gauges.go]: Would you lowercase? suggest `uptimeGauge`

**Review by rluisr — CHANGES_REQUESTED** [review]
Thank you for a pull request for enhancement!

File: trex/gauges.go (hunk 1 above)
Comment (warren-ru) [reply trex/gauges.go]: Done

**Review by rluisr — APPROVED** [review]
Thanks!

**Comment by rluisr** [comment]
released v1.2.0

**Pull request merged**